CSharpWithDebug/README.md

```diff
@@ -0,0 +1,10 @@
+### USAGE
+install [dotnet 8 sdk](https://dotnet.microsoft.com/en-us/download)
+```
+dotnet build
+```
+
+Start with option to debug
+```
+dotnet petooh.dll /path/to/file.ko [-d]
```

Comment:
I tried to use it, but `dotnet petooh.dll` is probably a platform-specific command. I found in the Mac OSX the binary file generated as `./bin/Debug/net8.0/petooh` that can be also used. Is it possible to generate something ready-to-use that will work the same way on each platform? Maybe a kind of `build` parameter or so. 🤔 

If not, then maybe it makes sense to change the running command to something like `~$ ./bin/Debug/net8.0/petooh ../test.koko [-d]` which will also work on every platform. But I'm not sure what the right way here is.

Reply:
my bad. On Mac (and any other platform with installed dotnet sdk) can be used with simple ```dotnet run ../test.koko -d```. Added this into README.md